autoload/test/swift/swiftpm.vim

```diff
@@ -25,16 +25,16 @@ function! test#swift#swiftpm#build_args(args) abort
   return a:args
 endfunction
 
-function test#swift#swiftpm#executable() abort
+function! test#swift#swiftpm#executable() abort
   return 'swift test'
 endfunction
 
-function! s:parse_module_info(position)
+function! s:parse_module_info(position) abort
   return s:get_first_match(a:position['file'], g:test#swift#patterns['module'])
 endfunction
 
-function! s:parse_case_info(position)
-  let l:result = ""
+function! s:parse_case_info(position) abort
+  let l:result ''
```

Comment:
There is a syntax typo here, it should be:

```vim
let l:result = ''
```

Reply:
Thanks for the review, fixed.